store/localcachelayer/file_info_layer_test.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package localcachelayer
+
+import (
+	"testing"
+
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mattermost-server/store/storetest"
+	"github.com/mattermost/mattermost-server/store/storetest/mocks"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestFileInfoStore(t *testing.T) {
+	StoreTest(t, storetest.TestFileInfoStore)
+}
+
+func TestFileInfoStoreCache(t *testing.T) {
+	fakeFileInfo := model.FileInfo{PostId: "123"}
+
+	t.Run("first call not cached, second cached and returning same data", func(t *testing.T) {
+		mockStore := getMockStore()
+		cachedStore := NewLocalCacheLayer(mockStore, nil, nil)
+
+		fileInfos, err := cachedStore.FileInfo().GetForPost("123", true, true, true)
+		require.Nil(t, err)
+		assert.Equal(t, fileInfos, []*model.FileInfo{&fakeFileInfo})
+		mockStore.FileInfo().(*mocks.FileInfoStore).AssertNumberOfCalls(t, "GetForPost", 1)
+		require.Nil(t, err)
```

Comment:
I don't understand why you are asserting ´err´ and ´fileInfos´again if they weren't assigned :thinking:

Reply:
Good, point, is a mistake, is not needed.